auto/src/glew_init_gl.c

```diff
@@ -5,8 +5,22 @@ GLboolean GLEWAPIENTRY glewGetExtension (const char* name)
   const GLubyte* start;
   const GLubyte* end;
   start = (const GLubyte*)glGetString(GL_EXTENSIONS);
```

Comment:
My branch checks to see if we have a 3.0+ context and always uses the new extension fetch mechanism.